tests/requirements.txt

```diff
@@ -28,7 +28,7 @@ pyflakes
 pytest
 pytest-cov
 pytest-mock
-pytest-rerunfailures
+pytest-rerunfailures<=11.0.0a0  # https://github.com/pytest-dev/pytest-rerunfailures/issues/206
```

Comment:
This might have been fixed: https://pypi.org/project/pytest-rerunfailures/11.1.1/#section-1

Reply:
Waiting on https://github.com/conda-forge/pytest-rerunfailures-feedstock/pull/12

Reply:
Was just merged